python/mypy.ini

```diff
@@ -33,3 +33,12 @@ disallow_incomplete_defs = True
 strict_equality = True
 warn_unreachable = True
 warn_unused_ignores = True
+
+[mypy-lsst.daf.butler.datastores.*]
+ignore_missing_imports = False
+ignore_errors = False
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+strict_equality = True
+warn_unreachable = True
+warn_unused_ignores = True
```

Comment:
Can you also turn mypy on for a few `core` modules after the changes you've made to them?  Or does that just bring new errors?